project/currency_comparison/__tests__/currency_comparison.test.js

```diff
@@ -0,0 +1,50 @@
+const CurrencyComparison = require('../currency_comparison')
+const axios = require('axios')
+const { test } = require('@jest/globals')
+jest.mock('axios')
+
+const testSalary = new CurrencyComparison(50000)
```

Comment:
I'm not sure if it's a good practice to have a constant value used across tests like this. In unit testing, each test should be self contained.